public/absences/js/modif.js

```diff
@@ -221,7 +221,7 @@ $(function() {
       var CSRFToken = $('#CSRFSession').val();
       var baseURL = $('#baseURL').val();
       var id=$("#absence-bouton-supprimer").attr("data-id");
-      document.location.href = baseURL + "/index.php?page=absences/delete.php&id="+id+"&CSRFToken="+CSRFToken;
+      document.location.href = baseURL + "/absence&id="+id+"&CSRFToken="+CSRFToken;
```

Comment:
/absence/id no ?

Reply:
/absence/id isn't the right route to me as we are deleting and not editing an absence.